static/app/components/stream/group.tsx

```diff
@@ -133,7 +133,8 @@ function BaseGroupRow({
   }, [organization, group.id, group.owners, query]);
 
   const trackClick = useCallback(() => {
-    if (query === Query.FOR_REVIEW) {
+    const forReviewQueries: string[] = [Query.FOR_REVIEW, Query.FOR_REVIEW_OLD];
```

Comment:
@17hogeju Nit: you could make this an export from `utils.tsx` to avoid having to redefine it so many times.